# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-alpha-bitness.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-46.4/python/mozbuild/mozbuild/configure/constants.py.vanilla	2026-02-06 20:52:50.076588545 +0100
+++ Arctic-Fox-46.4/python/mozbuild/mozbuild/configure/constants.py	2026-02-06 20:53:04.481201511 +0100
@@ -40,7 +40,7 @@
 
 CPU_bitness = {
     'aarch64': 64,
-    'Alpha': 32,
+    'Alpha': 64,
     'arm': 32,
     'hppa': 32,
     'ia64': 64,
